gridScale.js

```diff
@@ -997,11 +997,12 @@ caclByXnY(x,y) {
     let curScene = game.scenes.get(canvas.scene.data._id);     //This gets the scene object for the current scene by asking the canvas for the current scenes ID then reutrning that to the game.scenes.get
     //console.log("Grid Scale | Drawing Layer | ^^^^^ this is datacoords ^^^^^")
     console.log(sgLayer.dataCoords);
-    let gridPix = Math.floor(sgLayer.dataCoords[3]);      //getting the grid pixel size
+    let gridPix = sgLayer.dataCoords[3];      //getting the grid pixel size

```

Comment:
I removed the `Math.floor` instruction here because we aren't reading from `gridPix` yet. In the event the `sgLayer.currentTool` is `"3x3"`, it may be updated again with a division.

After all writes are complete, I added a `Math.round` instruction. My thinking behind this was so that if the `gridPix` came out to 69.999999 it wouldn't round down to 69, but round up to 70.